setup.cfg

```diff
@@ -55,7 +55,7 @@ python_requires = >=3.6.*
 
 install_requires =
     attrs >= 18.1, !=20.1.0
-    click >= 6.7, !=7.0
+    click >= 8.1.3, !=7.0
```

Comment:
Why do you need Click 8.1.3 + ?

Reply:
Because of #54 , I have referred same in commit message as well.